lang/semgrep-grammars/src/semgrep-java/test/corpus/semgrep.txt

```diff
@@ -185,7 +186,7 @@ class $X {
               (identifier))))))))
 
 ================================================================================
-Metavariable ellipsis parameter
```

Comment:
I think we do actually normally call this feature "metavariable ellipsis," not "named ellipsis."